views/index.pug

```diff
@@ -0,0 +1,65 @@
+html
+  head
+    title 2019 Stanley Cup Finals Stats API
+    link(rel='stylesheet', href='/css/styles.css')
+  body
+    header
+      .header 2019 Stanley Cup Finals Stats API
+    table
+        tr
+          td
+            .blank
+          th
+            text Get/teams
+          th
+            text Get/teams/{team}
+          th
+            text Get/players
+          th
+            text GET/player/{lastName}
+          th
+            text POST/player
```

Comment:
This route is impossible to call without looking at your code, there's no information here on what "player details" looks like, there is also no indication of the data format expected since you don't instruct what content-type header to use. Also the route is actually `/players` not `/player`